aiogram/types/message.py

```diff
@@ -95,10 +95,10 @@ def content_type(self):
             return ContentType.VOICE[0]
         if self.contact:
             return ContentType.CONTACT[0]
-        if self.location:
-            return ContentType.LOCATION[0]
         if self.venue:
```

Comment:
Since **Venue** contains `location` **and** `venue` properties, isn't it better to check for both of them, rather then rely on their positioning in the code?
`if self.venue and self.location`